packages/cadl-ranch-specs/http/type/property/value-types/mockapi.ts

```diff
@@ -91,3 +91,14 @@ Scenarios.Type_Property_ValueTypes_DictionaryString_put = passOnSuccess(dictiona
 const neverMock = createMockApis("never", undefined);
 Scenarios.Type_Property_ValueTypes_Never_get = passOnSuccess(neverMock.get);
 Scenarios.Type_Property_ValueTypes_Never_put = passOnSuccess(neverMock.put);
+
+const recordUnknownMock = createMockApis("record/unknown", {
```

Comment:
I think this test should go in the dictionary test, this is basically that.

Reply:
Oh, I see. There already exists a test case of `Record<unknown>` under dictionary. I will close this PR.

Reply:
yeah it is a bit different but the goal of this test is to test different types for a model property
while the other is to test the different types for the value of a record.

So in this test we want to test one Record but we don't need to test every single record combination on top.
This shouldn't make any difference in the generation no?

Reply:
Do you mean the we only need one Record<> test for type-property test and left tests for all kinds of types for Record value in dictionary test? If so, I'm OK to close this PR.

Reply:
yep, I don't think it brings extra value